<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsEditReservedWordsBase</class>
 <widget class="QDialog" name="QgsEditReservedWordsBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Reserved Words</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>10</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="Q3ListView" name="lvColumns" >
     <property name="defaultRenameAction" >
      <enum>Q3ListView::Accept</enum>
     </property>
     <column>
      <property name="text" >
       <string>Status</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Column name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Index</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="Q3TextEdit" name="txtExplanation" >
     <property name="maximumSize" >
      <size>
       <width>32676</width>
       <height>75</height>
      </size>
     </property>
     <property name="text" >
      <string>This shapefile contains reserved words. These may affect the import into PostgreSQL. Edit the column names so none of the reserved words listed at the right are used. You may also change any column name if desired.</string>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonHelp" >
       <property name="text" >
        <string>&amp;Help</string>
       </property>
       <property name="shortcut" >
        <string>F1</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <widget class="Q3GroupBox" name="groupBox1" >
     <property name="title" >
      <string>Reserved Words</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>10</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="Q3ListBox" name="lstReservedWords" >
        <property name="maximumSize" >
         <size>
          <width>120</width>
          <height>32767</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListView</class>
   <extends></extends>
   <header>q3listview.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends></extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends></extends>
   <header>q3textedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends></extends>
   <header>q3listbox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
